content/kotsadm/installing/stateful-components.md

```diff
@@ -7,69 +7,40 @@ weight: 10065
 
-## Postgresql
+# Postgresql
 
 KOTS uses a Postgresql StatefulSet to store the version history, application metadata and other small amounts of data needed to manage the application(s).
 This Postgresql component is deployed with KOTS and secured with a randomly generated password, and only exposed as a ClusterIP on the overlay network.
 
-{{< notes title="Future Plans">}}
+# S3-Compatible Object Store
 
-A future version of KOTS will remove the Postgresql requirement to enable multiple replicas of every component and facilitate a true highly-available deployment.
-The work is being tracked in [GitHub issue #537](https://github.com/replicatedhq/kots/pull/537).
+By default, KOTS also requires an S3-compatible object store for application archives and support bundles. KOTS can be deployed and upgraded to remove this requirement, as described below. 
 
-{{< /notes >}}
+## Embedded Clusters
 
-## S3 Compatible Object Store
-
-KOTS also requires an S3 compaitble object store to store application archives and support bundles.
-A KOTS installation to an existing cluster will deploy Minio to satisfy this requirement, and nothing is required during installation.
-When deploying, Minio is configured with a randomly generated AccessKeyID and SecretAccessKey, and only exposed as a ClusterIP on the overlay network.
-
-{{< notes title="Future Plans">}}
-
-A future version of KOTS will remove compatibiltiy with S3 in favor of a unified storage backend using OCI registries.
-KOTS 1.19 contains the first release of this.
-Before S3 is removed, full compatibility will be guaranteed and KOTS will be able to seemlessly and automatically migrate data.
-
-{{< /notes >}}
-
-## OCI Registry (alpha support)
-
-Beginning with KOTS 1.19, experimental support has been added to use an OCI Registry for storage instead of the Object Store listed above.
-When this feature is enabled, KOTS will deploy [docker distribution](https://github.com/docker/distribution) to satisfy the storage requirement.
+By default, embedded cluster installers must include a add-on that satisfies the S3-compatible object store.
+This could be the Rook or Minio add-ons.
+Without one of these add-ons, the installer will fail with an error.
 
+Embedded Clusters can have optional, indirect dependencies on the S3-compatible object store. 
+1. The Registry add-on, used for air-gapped, embedded installs, will use object storage if available. Otherwise it will use the default Storage Class.
+1. The Velero add-on, used for snapshots, will default to saving snapshots in the object store (i.e. the **Internal** option in the console).
 
-When using the OCI Registry instead of the S3 compatible object store, a few benefits are realized:
+To deploy an embedded cluster without object storage, set the `disableS3` flag to `true` in the KOTS add-on.
+This will deploy KOTS without object storage, as well as allow the supporting add-ons to use Persistent Volumes (PVs) instead of object storage.
```

Comment:
```suggestion
This will deploy KOTS without an object storage, as well as allow the supporting add-ons to use Persistent Volumes (PVs) instead of object storage.
```

Reply:
Same question as above.